### PR TITLE
Remove privilege escalation.

### PR DESCRIPTION
**Pull request by rabbah**
I'm not sure why the privilege escalation is needed - suggesting we remove them.

PG2/1868 🔵 

@jeremiaswerner @markusthoemmes @csantanapr could you sanity check please.

### CODE REVIEW

**Comment by rabbah** [comment]
hmm - maybe travis just told me the answer.

**Comment by rabbah** [comment]
If privilege escalation is needed, one can recover this with `ansible-playbook --become-user=root`.

**Comment by csantanapr** [comment]
Another alternative is the effective user running the `ansible-playbook` can have the high privilege, I vote not to have the privilege inside the scripts.

File: tools/travis/build.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ set -e
 SCRIPTDIR=$(cd $(dirname "$0") && pwd)
 ROOTDIR="$SCRIPTDIR/../.."
 HOMEDIR="$SCRIPTDIR/../../../"
+export PYTHONPATH=/opt/python/2.7.13/lib/python2.7/site-packages
```
Comment (markusthoemmes) [inline tools/travis/build.sh]: Why's this needed?

Comment (rabbah) [inline tools/travis/build.sh]: there are two versions of python installed per the travis logs (3.x and 2.7) this forces the latter to be used.

**Comment by rabbah** [comment]
rebased to pick up consul change.

**Pull request merged**